src/functions/build_dependency_packages.py

```diff
@@ -20,19 +20,38 @@ def build_dependency_packages(mpr_rpc_json_data, resultcount):
 
 		depends_packages = []
 
-		for j in ['depends', 'makedepends', 'checkdepends']:
+		if j == (subprocess.check_output(["lsb_release", "-cs"]) + '_depends'):
 			depends_packages += get_srcinfo_value(j, True)
+		else:
+			for j in ['depends', 'makedepends', 'checkdepends']:
+				depends_packages += get_srcinfo_value(j, True)
```

Comment:
This isn't going to act correctly if something like `focal_makedepends` is set, but `focal_depends` isn't. You also appear to only be setting dependencies from `distro_depends` and not `distro_makedepends` and `distro_checkdepends`.